angelsbioprocessing/locale/en/bio-processing.cfg

```
@@ -1,538 +1,538 @@
-bio-refugium-biter=Biter Refugium
-bio-refugium-hogger=Hogger Refugium
-
-[item-description]
-composter="Used to turn any Organic Material into compost"
-solid-saw="He saw, he cut and he shattered"
-
-temperate-1="Wil. Plant from a place where no man has gone before"
-temperate-2="Fluffy. Makes you want to go to bed"
-temperate-3="Sour. Best served in a hot sauce"
-temperate-4="Adaptive. Can be used for a variety of things"
-temperate-5="Tasty. Very generous range of flavours"
-
-desert-1="Sweet. Gets you all fuzzy and warm around your heart"
-desert-2="Organized. Grows in ordered lines"
-desert-3="Productive. Will always arrange itself in orderly lines"
-desert-4="Zealous. Always dynamic"
-desert-5="Salty. Has to be handled with care"
-
-swamp-1="Nutty. Makes you slightly mad"
-swamp-2="Lively. Seem to be growing disregarding the conditions"
-swamp-3="Commanding. Don't use too much"
-swamp-4="Binary. It grows or it does not, nothing in between"
-swamp-5="Spicy. Seems to yell at you"
-
-solid-nuclear-fertilizer="This doesn't sound healty... We might start to glow at night!"
-
-raw-fish="A friendly reminder to who knew Factorio before an Angel stopped by"
-alien-fish-1-raw="The least deadly fish on this planet"
-alien-fish-2-raw="Oblivious of their surroundings, very frugal beings"
-alien-fish-3-raw="Lovely Fish at first sight, but don't anger them"
-
-bio-raw-meat="It tastes like chicken..."
-
-[entity-description]
-bio-refugium-biter="Fenced in safely from the outside world, biters can be grown here"
-bio-refugium-fish="More than just decoration, can be made home to a variety of fish"
-bio-refugium-hogger="Dirt digging Hogger feel right at home here"
-bio-refugium-puffer="It stinks, but Puffers feel right at home in this gas filled tower"
-
-[recipe-description]
-temperate-garden-a=Temperate Seed Extraction
-temperate-garden-b=Advanced Temperate Seed Extraction
-
-desert-garden-a=Desert Seed Extraction
-desert-garden-b=Advanced Desert Seed Extraction
-
-swamp-garden-a=Swamp Seed Extraction
-swamp-garden-b=Advanced Swamp Seed Extraction
+[entity-name]
+algae-farm=Algae farm 1
+algae-farm-2=Algae Farm 2
+algae-farm-3=Algae farm 3
+
+crop-farm=Basic farm
+temperate-farm=Farm with temperate upgrade
+desert-farm=Farm with desert upgrade
+swamp-farm=Farm with swamp upgrade
+
+composter=Composter
+seed-extractor=Seed extractor
+bio-press=Press
+bio-processor=Bio processor
+nutrient-extractor=Nutrient extractor
+
+temperate-garden=Temperate garden
+desert-garden=Desert garden
+swamp-garden=Swamp garden
+
+temperate-tree=Temperate tree
+swamp-tree=Swamp tree
+desert-tree=Desert tree
+
+puffer-nest=Puffer nest
+
+bio-arboretum-1=Arboretum
+
+bio-generator-temperate-1=Temperate tree seed generator
+bio-generator-swamp-1=Swamp tree seed generator
+bio-generator-desert-1=Desert tree seed generator
+
+bio-refugium-biter=Biter refugium
+bio-refugium-fish=Fish tank
+bio-refugium-hogger=Hogger refugium
+bio-refugium-puffer=Puffer refugiume
```

I think this is actually a typo
```suggestion
bio-refugium-puffer=Puffer refugium
```

---

Yes